ethpm/deployments.py

```diff
@@ -12,6 +18,11 @@
     Contract,
 )
 
+try:
+    from typing import TypedDict  # type: ignore
+except ImportError:
+    from mypy_extensions import TypedDict  # <=3.7
+
```

Comment:
@pipermerriam I'm not sure if this is the best way to handle this, since `web3` will be supporting 3.8 soon, where `TypedDict` is automatically included with `typing`? Is it better to just go with the `mypy_extensions` import for now and add `mypy_extensions` to `setup.py['install_requires']`?

Reply:
This is generally an acceptable pattern.  If you find yourself copying these import lines across multiple files I'd recommend putting it somewhere like `library._compat` and then importing from there everywhere else.